File: apps/sdkwork-appstore-mini-program/packages/sdkwork-appstore-mp-games/src/routes/routeContributions.ts
import type { SdkworkUiRouteContribution } from "@sdkwork/appstore-mp-core";

/**
 * Route contributions for the games capability.
 *
 * Route ids are shared with the PC, H5, Flutter, and HarmonyOS roots
 * (`APP_CLIENT_ARCHITECTURE_ALIGNMENT_SPEC.md` section 7). The mini program
 * resolves each id through its own `pages` or `subpackages` map.
 */
export const gamesRouteContributions: readonly SdkworkUiRouteContribution[] = [
  { id: 'app.store.games.index', surface: 'app', domain: 'store', capability: 'games', screen: 'index', path: '/games', titleKey: 'appstore.games.index.title', auth: 'public', presentation: {"pc":"page","h5Mobile":"stack","flutterMobile":"route","miniProgram":"page","harmonyNative":"page"} }
] as const;

File: apps/sdkwork-appstore-flutter-mobile/packages/sdkwork_appstore_flutter_mobile_games/lib/src/routes/route_contributions.dart
import 'package:sdkwork_appstore_flutter_mobile_core/sdkwork_appstore_flutter_mobile_core.dart';

/// Route contributions for the games capability.
///
/// Route ids are shared with the PC, H5, mini program, and HarmonyOS roots
/// (`APP_CLIENT_ARCHITECTURE_ALIGNMENT_SPEC.md` section 7). Flutter maps each
/// id to its own named route.
const List<SdkworkUiRouteContribution> gamesRouteContributions =
    <SdkworkUiRouteContribution>[
  SdkworkUiRouteContribution(
    id: 'app.store.games.index',
    surface: SdkworkRouteSurface.app,
    domain: 'store',
    capability: 'games',
    screen: 'index',
    path: '/games',
    titleKey: 'appstore.games.index.title',
    auth: 'public',
  ),
];
